runtime/colors/tools/check_colors.vim

```diff
@@ -1,238 +1,209 @@
-" This script tests a color scheme for some errors and lists potential errors.
-" Load the scheme and source this script, like this:
-"    :edit colors/desert.vim | :so colors/tools/check_colors.vim
+vim9script
+# This script tests a color scheme for some errors and lists potential errors.
+# Load the scheme and source this script, like this:
+#    :edit colors/desert.vim | :ru colors/tools/check_colors.vim
 
-let s:save_cpo= &cpo
+var save_cpo = &cpo
 set cpo&vim
 
-func! Test_check_colors()
-  let l:savedview = winsaveview()
-  call cursor(1,1)
-  let err = {}
+def Test_check_colors()
+    var savedview = winsaveview()
```

Comment:
Variables that don't change can be declared constant. Vim supported this even before `vim9script`.

Writing
```vim
const savedview = winsaveview()
```
shows the clear intention of the variable. Plus it catches errors early. It protects you from accidentally changing variables that aren't intended be changed. In some languages it's called _const-correctness_.

Same can be done for the variables `hi_groups`, `bg_set`, `bg_let`, `bg_pat`, `line`  below.

Reply:
Fixed. Could you check?